docker_config/dockerfiles/Dockerfile.default-gpu

```diff
@@ -64,23 +51,10 @@ RUN wget http://scala-lang.org/files/archive/scala-$SCALA_VERSION.deb && \
 ENV SCALA_HOME /usr/share/java
 
 ## Python packages
-# Newer pip dropped the support for python 2 so we need to specify the version.
-RUN python -m pip install --upgrade "pip < 21.0"
+
+## Python packages
```

Comment:
Remove this line (duplicated from above)

Reply:
Thanks, it has been removed.